versions/3.1.1.md

```diff
@@ -1882,27 +1899,24 @@ requestBody:
                 type: integer
 ```
 
-`application/x-www-form-urlencoded` is a text format, which requires base64-encoding any binary data:
+###### Example: Multipart Form with Multiple Files
 
-```YAML
+In accordance with [RFC7578 §4.3](https://www.rfc-editor.org/rfc/rfc7578.html#section-4.3), multiple files for a single form field are uploaded using the same name (`file` in this example) for each file's part:
+
+```yaml
 requestBody:
   content:
-    application/x-www-form-urlencoded:
+    multipart/form-data:
       schema:
-        type: object
         properties:
-          name:
-            type: string
-          icon:
-            # default for type string is text/plain, need to declare
-            # the appropriate contentType in the Encoding Object
-            type: string
-            contentEncoding: base64url
-  encoding:
-    icon:
-      contentType: image/png, image/jpeg
+          # The property name 'file' will be used for all files.
+          file:
+            type: array
+            items: {}
 ```
 
+As seen in the section on `multipart/form-data` below, the empty schema for `items` indicates a media type of `application/octet-stream`.
```

Comment:
Which section below? Appendix C or E?

Better add a link to the relevant section, and adjust "below" to "above" if needed.

Reply:
That text was left over from when things were in a different order - it's not about an appendix at all.  I'll sort it out and update - good catch!